webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9790,16 +9794,18 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                         var tabId = $mck_message_inner.data('mck-id');
                         if (resp.message.metadata.KM_STATUS) {
                             var groupId = 'li-group-' + resp.message.groupId;
+                            let conversationStatus = resp.message.metadata.KM_STATUS;
+                            MCK_GROUP_MAP && MCK_GROUP_MAP[resp.message.groupId] && (MCK_GROUP_MAP[resp.message.groupId].metadata.CONVERSATION_STATUS = (Kommunicate.conversationHelper.status[conversationStatus.toUpperCase()]));
+                            CURRENT_GROUP_DATA.tabId = resp.message.groupId;
+                            CURRENT_GROUP_DATA.conversationStatus = Kommunicate.conversationHelper.status[conversationStatus.toUpperCase()];
```

Comment:
@Kusum-Dhoundiyal Changing the conversationStatus to upperCase won't break the current flow, right?

Reply:
@ab14bhardwaj No this will not break anything. Actually we have map having keys upper case.Here we are just fetching value of status from map.
 status:{
        "INITIAL": -1,
        "OPEN": 0,
        "PROGRESS": 1,
        "CLOSED": 2,
        "SPAM": 3,
        "DUPLICATE": 4,
        "ARCHIVE": 5,
        "UNRESPONDED": 6,
        "WAITING":7
    }